### PR TITLE
Fix stylelint errors in facility-locator

### PR DESCRIPTION
**Pull request by bkjohnson**
## Description

This is a follow-up to #17305 . It fixes the `stylelint` errors in the `facility-locator` app.

## Testing done

- ` npx stylelint "src/applications/facility-locator/**/*.scss"`

## Screenshots

Before:

image

## Acceptance criteria
- [x] No `stylelint` errors in `facility-locator`

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 1]
```diff
@@ -477,47 +471,6 @@ $facility-locator-shadow: rgba(0, 0, 0, 0.5);
   background-color: $color-red-dark;
 }
 
-#infoBubble {
```
Comment (bkjohnson) [inline src/applications/facility-locator/sass/facility-locator.scss:480]: I couldn't find any DOM element with the `id` `infoBubble`. It was [added ~2.5 years ago](https://github.com/department-of-veterans-affairs/vets-website/pull/8651/files#diff-8dc9f9c094fb05328f951d321b0c6bd039a9dbbe990833d23a39b0a0a0326727R86) and got deleted somewhere along the way, but the CSS was left behind.

image

File: src/applications/facility-locator/sass/facility-locator.scss [hunk 2]
```diff
@@ -3,12 +3,6 @@
 @import "~leaflet/dist/leaflet.css";
 @import "~react-tabs/style/react-tabs";
 
-// TODO: These colors are not close enough to anything that already exists in
-// Formation. Perhaps the design should be reevaluated to see if Formation
-// colors could be used in place of these
-$facility-locator-color-info-bubble-border: #6495a3;
-$facility-locator-color-info-bubble-text: #818a8d;
-$facility-locator-color-gray: #ccc;
```
Comment (bkjohnson) [inline src/applications/facility-locator/sass/facility-locator.scss:11]: The gray color wasn't used at all, and the other two were only used as descendants of the unused `#infoBubble` selector

**Review by bkjohnson — COMMENTED** [review]
I did some additional cleanup beyond removing the hex literals.

**Review by mpelzsherman — APPROVED** [review]
Tested locally - LGTM!

**Pull request merged**